Sources/Views/ALKInformationCell.swift

```diff
@@ -76,8 +76,35 @@ final class ALKInformationCell: UITableViewCell {
 
     func update(viewModel: ALKMessageViewModel) {
         self.viewModel = viewModel
+        
+        //adding rating icons to the label
+        var ratingImage = UIImage()
+        guard let feedbackString =  viewModel.metadata? ["feedback"] as? String else { return }
+        do {
+            let feedbackDictionary = try? JSONSerialization.jsonObject(with: feedbackString.data(using: .utf8)!, options: []) as? [String:Int]
+            switch feedbackDictionary!["rating"]! {
+            case 1:
+                ratingImage = UIImage(named: "sadEmoji", in: Bundle.applozic, compatibleWith: nil)!
+            case 5:
+                ratingImage = UIImage(named: "confusedEmoji", in: Bundle.applozic, compatibleWith: nil)!
+            case 10:
+                ratingImage = UIImage(named: "happyEmoji", in: Bundle.applozic, compatibleWith: nil)!
+            default:
+                print("incoreect data")
+            }
```

Comment:
Create a separate function to get the rating image based on the value. I guess we already have it, so check once.

Reply:
We do not have the function in Applozic

Reply:
Then just do this:
> Create a separate function to get the rating image based on the value.